vignettes/reporting.rmd

```diff
@@ -0,0 +1,394 @@
+---
+title: "New reporting in teal"
```

Comment:
I would love to keep our docs timeless and would recommend to avoid time-related phrases - even in the title. For example, if I open this document in a year or so - what does "new" actually mean?
Please just focus on explaining the current state as in typical technical manual. If we need to explain the deltas between the versions - that should be a separate release related entity (article, blog post, supplementary document etc.). As an example, we have done [this](https://github.com/insightsengineering/teal/discussions/988) in the past.

Reply:
also, title shouldn't differ from a filename.